lpcr_donation_tax_report/models/res_partner.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2025 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_address_valid = fields.Boolean(
+        string="Complete info",
```

Comment:
```suggestion
        string="Complete Info",
```
field descriptions should use title case.